tests/script-based-pre/concrete_playback_test/bin/src/main.rs

```diff
@@ -0,0 +1,29 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+fn main() {}
+
+#[cfg(kani)]
+mod harnesses {
+    #[kani::proof]
+    fn harness() {
+        let result_1: Result<u8, u8> = kani::any();
+        let result_2: Result<u8, u8> = kani::any();
+        assert!(!(result_1 == Ok(101) && result_2 == Err(102)));
+    }
+
```

Comment:
Can we add a reference to the `ui` sibling of this test?

Reply:
That's a good idea. I'll add a comment referencing the original UI test.